src/bronto_burt.c

```diff
@@ -0,0 +1,900 @@
+#include "bronto_burt.h"
+#include "functions.h"
+#include "object.h"
+#include "task.h"
+
+struct Object2* CreateBrontoBurt(struct Object* arg0, u8 arg1) {
+    struct Object2 *obj, *obj2;
+    struct Task *task = TaskCreate(ObjectMain, 0xb4, 0x1000, 0x10, ObjectDestroy);
+    obj = TASK_GET_STRUCT_PTR(task, obj2);
+    InitObject(obj, arg0, arg1);
+    if(obj->x > obj->kirby3->base.x) {
```

Comment:
code style: space after control-flow statement reserved keywords (`if` `while` `for` `switch`)